dev-php/ca-bundle/ca-bundle-1.0.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Find a path to the system CA bundle, and fallback to the Mozilla CA bundle"
+HOMEPAGE="https://github.com/composer/ca-bundle"
+
```

Comment:
Is there a reason why you add this empty line? We generally try to preserve the order and grouping variables from the devmanual, to help readability, and this bites me in the eyes.